api/src/main/java/org/apache/cloudstack/backup/BackupManager.java

```diff
@@ -38,7 +38,7 @@
     ConfigKey<Boolean> BackupFrameworkEnabled = new ConfigKey<>("Advanced", Boolean.class,
             "backup.framework.enabled",
             "false",
-            "Is backup and recovery framework enabled.", true, ConfigKey.Scope.Zone);
+            "Is backup and recovery framework enabled. Restart management server on global value change.", true, ConfigKey.Scope.Zone);
```

Comment:
According to the description, ADMIN needs to restart the management server and therefore this is not a dynamic setting. 

What do you think of changing the `isDynamic` param to `false`?

_ConfigKey constructor:_
```
public ConfigKey(String category, Class<T> type, String name, String defaultValue, String description, boolean isDynamic, Scope scope) {
    this(type, name, category, defaultValue, description, isDynamic, scope, null);
}
```

Reply:
I'll change that.

Reply:
Fixed thanks